test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -581,74 +580,6 @@
           (is (= body msg))
           (is (= status http/status-bad-request)))))))
 
-(deftestseq ^{:postgres false} two-database-fact-query-config
```

Comment:
I think we need to port the two-database handling to FOSS so we can preserve this. We need coverage of the two database scenario in core as well.